parts/k8s/kubernetescustomscript.sh

```diff
@@ -281,6 +281,24 @@ function configNetworkPlugin() {
     fi
 }
 
+function installKataContainersRuntime() {
```

Comment:
I realize that my comments in this function also apply to `installClearContainersRuntime()`, which is probably why you did it that way but I think `installClearContainersRuntime` needs to be changed as well to have proper exit codes.

Reply:
No worries.  Okay to update the example function, `installClearContainersRuntime()`, in a follow-on PR?